docs/BestPractices.md

```diff
@@ -23,6 +23,13 @@ FROM node:4.1.2-onbuild
 RUN groupadd -r node && useradd -r -g node node
 ```
 
+When using the `alpine` variant, add the user like so:
+
+```Dockerfile
+FROM node:7.3.0-alpine
+RUN addgroup -S node && adduser -S -g node node 
+```
```

Comment:
Accidental necrobump on my part—sorry about that.

Appears to be correct in #307, and already moved on from in head anyway, so disregard. :grinning: